internal/cli/server.go

```diff
@@ -95,17 +108,15 @@ func logRequestHandler(h http.Handler, ui *termui.UI) http.Handler {
 	return http.HandlerFunc(fn)
 }
 
-func logRequest(r *http.Request, ui *termui.UI) {
+func logRequest(r *http.Request, log logr.Logger) {
 	uri := r.URL.String()
 	method := r.Method
-	ui.Normal().V(1).Msgf("%s %s", method, uri)
-
-	msg := ui.Normal().V(1).Compact()
+	log.V(1).Info(fmt.Sprintf("%s %s", method, uri))
```

Comment:
With a logger, should we split the data into proper fields for the log message ?

Reply:
I'm not going to change this as part of this PR.
I'm not sure what these fields would look like or how you'd configure them with logr. 
Someone has to convince me of Verbosity-levels and structured logging. 

Maybe we can have a story to make API server request logging better.

Reply:
Yes, we can do that in a separate ticket.